nico/backends/js.nim

```diff
@@ -686,15 +686,13 @@ proc getMusic*(channel: int): int =
   return -1
 
 proc music*(channel: AudioChannelId, musicId: MusicId, loop: int = -1) =
-  if audioContext == nil:
```

Comment:
Is there a reason for removing this check?
This is here so that if the audio context couldn't be created all the audio procs will be no-ops rather than trigger errors when they touch objects created from the audioContext.